resources/metadata/nexusUpload.yaml

```diff
@@ -110,7 +110,7 @@ spec:
           - STEPS
         aliases:
           - name: maven/m2Path
-      - name: user
+      - name: username
```

Comment:
```suggestion
      - name: username
        aliases:
          - name: user
            deprecated: true
```

Reply:
Not sure this is needed. The alias has no effect for command line parameters. It would only apply to config, or to env-variables (PIPER_user and PIPER_username would both work). But in which situations would this be needed? The Groovy wrapper would always be at least as recent as the piper binary, correct?

Reply:
Do you think someone could be supplying the username via their config? I thought they would always supply the Jenkins credentials ID.

Reply:
Ok, you are probably right.